packages/getting-started/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@loopback/getting-started",
+  "version": "1.0.0",
+  "description": "An application and tutorial on how to build with LoopBack 4.",
+  "main": "index.js",
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
```

Comment:
Can we add `"private" : true,` so that the module won't be published to npmjs? See https://github.com/strongloop/loopback-next/blob/master/packages/example-codehub/package.json#L8

Reply:
I think we want this one to be published to npmjs, so that people can install it as a ready-made example, don't we?